client/blocks/login/login-form.jsx

```diff
@@ -271,7 +277,7 @@ export class LoginForm extends Component {
 							{ this.props.translate( 'Not on WordPress.com? {{signupLink}}Create an Account{{/signupLink}}.',
 								{
 									components: {
-										signupLink: <a href={ oauth2ClientData.signupUrl } />,
+										signupLink: <a href={ signupUrl } />,
```

Comment:
The only problem with doing this here is it means we can't merge this PR until the styles are ready. I think we should remove this change until everything else is ready.